src/main/java/AlexLink/HomeWork/HomeworkJDBS/DAO/SelectMoviesDAO.java

```diff
@@ -0,0 +1,12 @@
+package AlexLink.HomeWork.HomeworkJDBS.DAO;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+public interface SelectMoviesDAO {
+    List<String> SelectTitleOfMoviesByCurrentYear();
```

Comment:
Method name from uppercase letter

Reply:
fixed

Reply:
Added the structure of MySql, if I understood rightly.

Reply:
Added a new method which change year of release of the movie by title